torch/_dynamo/variables/misc.py

```diff
@@ -462,9 +358,11 @@ def reconstruct(self, codegen):
         ).reconstruct(codegen)
 
         if codegen.tx.output.partial_convert:
+            loads = [codegen.create_load_const(val) for val in self.ctx.target_values]
+            output.extend(loads)
             output.extend(
                 [
-                    create_instruction("CALL_FUNCTION", 0),
+                    create_instruction("CALL_FUNCTION", len(loads)),
```

Comment:
Yes, but it is not sufficient; our code for `reconstruct`-ing the `SETUP_WITH` variables on the stack after tracing out the subgraph with `compile_subgraph` is erroneous as we do not load any arguments to the function or call it.

It will be fixed in subsequent PR.

Reply:
Yea, this is a known issue, glad if your can fix.